### PR TITLE
Prepare for publishing

### PR DESCRIPTION
**Pull request by lmelninkas**
- Adds GHA wokflows for publishing to PyPI
- Updates project setup

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,14 +1,24 @@
 [tool.poetry]
-name = "pyap"
-version = "0.3.1"
-description = "Pyap is an MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses."
-authors = ["Vladimir Goncharov <vladimarius@gmail.com>"]
-documentation = "https://github.com/vladimarius/pyap"
+name = "argyle-pyap"
+version = "0.0.0"
+description = "Argyle's Pyap is a fork of Pyap, a MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses."
```
Comment (DeviousStoat) [inline pyproject.toml:4]: maybe just adding a small comment about why this fork. Like `is a fork of Pyap` ... `with a few improvements and supporting a lot more addresses formats`

File: pyproject.toml [hunk 2]
```diff
@@ -1,14 +1,24 @@
 [tool.poetry]
-name = "pyap"
-version = "0.3.1"
-description = "Pyap is an MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses."
-authors = ["Vladimir Goncharov <vladimarius@gmail.com>"]
-documentation = "https://github.com/vladimarius/pyap"
+name = "argyle-pyap"
```
Comment (DeviousStoat) [inline pyproject.toml:2]: Maybe we could add
https://python-poetry.org/docs/pyproject/#keywords
https://python-poetry.org/docs/pyproject/#homepage
https://python-poetry.org/docs/pyproject/#repository

**Comment by lmelninkas** [comment]
Published at PyPI test https://test.pypi.org/project/argyle-pyap/0.0.0a137/

**Comment by lmelninkas** [comment]
published at pypi test https://test.pypi.org/project/pyap2/0.0.0a142/

**Pull request merged**